Frizk - not askin 4 much

tposejank opened the pull request:
## Chart Verification PR

<!-- Here you should change these to the artist and song names. -->

Frizk - not askin 4 much

<!-- If possible, upload the album art below. -->

![cover](https://github.com/Encore-Developers/songs/assets/81495861/913d9c3c-54c0-4a78-9f13-0c414105f2d9)

### Instruments and difficulties charted

<!-- Denote all difficulties and charts you have charted, so it's easier 
for the verifiers to verify the chart.
You can do this by changing "No" to "Yes", if you charted it. -->

| Instrument     | Easy     | Medium    | Hard      | Expert       |
| -------------- | -------- | --------- | --------- | ------------ |
| Drums          | No       | No        | No        | Yes           |
| Bass           | No       | No        | No        | Yes           |
| Lead           | No       | No        | No        | Yes           |
| Vocals         | No       | No        | No        | Yes           |
| Pro Drums      | No       | No        | No        | Yes           |
| Pro Bass       | No       | No        | No        | Yes           |
| Pro Lead       | No       | No        | No        | Yes           |

### Additional details

<!-- If there are any details about your chart, like for example:
intentionally placed invalid chords, no lifts or something else,
you can specify below to make the verifier aware of it. -->

Extended sustains in vocals at 1:12, hes singing chords so i did that

NarrikSynthfox commented:
**All note times given are in Measures.Beats time format for REAPER**

Drums:

- The lane 5 note at 5.4 could be moved slightly back to 5.3.81 (1/32 dotted snap)
- In the intro and outro, the pattern of alternating between lane 3 and 4 is very uncomfortable to play on a controller, maybe switch it to start on lane 4 and alternate between 4 and 2?

Bass: The bass is very simplified compared to what is actually being played, and probably should be recharted.

Lead:

- The pattern of taps and lifts for the chords is off slightly.
- At 38.2.5 and 46.2.75, the alternating pattern between lanes 4 and 5 is uncomfortable to play on controller and should be changed so that it would switch hands. Also the one at 38.2.5 isn't alternating the same pitch so possibly that should be reworked.
- Some of the sustains should be shortened slightly so that there's a gap between them, because otherwise the player is forced to let go of them early to switch to the next note on controller

Vocals:
- The lifts during the chorus aren't really needed, it actually feels better to tap the note instead
- The extended sustains are weird to hit with a controller as well as not accurate to what actually is going on in the vocal, I would chart it as 2 note chords. The rhythm for the vocals sounds like it would be something like this (A vertical line denotes a measure line and an x would be where a note would be placed): |--x---x-x-x-x---|--x--xx-x-x-x---|

tposejank commented:
> Drums:
> 
> * The lane 5 note at 5.4 could be moved slightly back to 5.3.81 (1/32 dotted snap)

Fixed, and added another lane 5 shortly before one lane 3

> * In the intro and outro, the pattern of alternating between lane 3 and 4 is very uncomfortable to play on a controller, maybe switch it to start on lane 4 and alternate between 4 and 2?

Changed that layout to lane 2 & 4, 4^ and lane 2 & 5

> Bass: The bass is very simplified compared to what is actually being played, and probably should be recharted.

Recharted and using correct pitching now

> Lead:
> 
> * The pattern of taps and lifts for the chords is off slightly.

Fixed in [10d7566](https://github.com/Encore-Developers/songs/pull/2/commits/10d7566e4bb9ff306fa58819c0ad04ed0d9d04ac) by removing the last lift at each chord (thats what felt wrong)

> * At 38.2.5 and 46.2.75, the alternating pattern between lanes 4 and 5 is uncomfortable to play on controller and should be changed so that it would switch hands. Also the one at 38.2.5 isn't alternating the same pitch so possibly that should be reworked.

Fixed alternating notes and changed pitches to more accurate patterns, while accounting for thumb placements

> * Some of the sustains should be shortened slightly so that there's a gap between them, because otherwise the player is forced to let go of them early to switch to the next note on controller

Added sustain gaps

> Vocals:
> 
> * The lifts during the chorus aren't really needed, it actually feels better to tap the note instead

Removed the lifts in the chorus. That was true

> * The extended sustains are weird to hit with a controller as well as not accurate to what actually is going on in the vocal, I would chart it as 2 note chords. The rhythm for the vocals sounds like it would be something like this (A vertical line denotes a measure line and an x would be where a note would be placed): |--x---x-x-x-x---|--x--xx-x-x-x---|

Removed those. Only taps there

NarrikSynthfox commented:
Drums are fine now

Bass, the first sustain of each measure should be turned into a tap
The sustains in the intro on lead should be lengthened by 1/16th to match the outro 

For vocals, the chorus probably should be changed slightly to follow the pitch a little better, as well as the 2 3 4 5 pattern in the mid section of the song doesn't follow what's actually going on there
